package/rebuild-todo

```diff
@@ -214,6 +216,7 @@ for pkg in "${packages[@]}"; do
             SKIP_BUILD=0
             while true; do
                 if pkgctl build --rebuild $REPO; then
+		    rebuilt_packages+=("$pkg")
```

Comment:
This script uses spaces for indentation. I'm not too crazy about tabs but can handle them just fine too as long as the whole file/project is consistent. But for the love of pete, please don't mix tabs and spaces for indentation on the same line!

Reply:
Uh... This is not intentional, probably a messed up copy paste or something :shrug: Fixed https://github.com/archlinux/contrib/pull/74/commits/9d9f608eafb4e13686c1a37ef5e88ac09b57d6ca :wink: